cli/command/image/build_test.go

```diff
@@ -18,6 +20,52 @@ import (
 	"golang.org/x/net/context"
 )
 
+func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Invalid test on Windows")
+	}
+	dest, err := ioutil.TempDir("", "test-build-context-dest")
+	require.NoError(t, err)
+	defer os.RemoveAll(dest)
```

Comment:
We have helpers for this: https://godoc.org/github.com/gotestyourself/gotestyourself/fs

```go
dest := fs.NewDir(t, "test-build-context-dest")
defer dest.Remove()

// use dest.Path() to get the path
```